Tribler/Test/Community/Multichain/statistics/test_page_rank.py

```diff
@@ -1,3 +1,6 @@
+"""
+Tests for the incremental PageRank.
+"""
 from unittest import TestCase
 
 from networkx import gnm_random_graph
```

Comment:
@MaxVanDeursen NetworkX is actually still used. See for example our [incremental pagerank](https://github.com/vandenheuvel/tribler/blob/master/Tribler/community/multichain/statistics/page_rank.py#L70) implementation.